google/cloud/bigtable/table_admin.cc

```diff
@@ -922,6 +922,31 @@ StatusOr<google::iam::v1::Policy> TableAdmin::GetIamPolicy(
   return proto;
 }
 
+StatusOr<google::iam::v1::Policy> TableAdmin::GetIamPolicy(
```

Comment:
we should write unit tests in `table_admin_test.cc`.

we should also write integration tests in `admin_iam_policy_integration_test.cc`, it can be a todo for now and can go in separate PR.